compiler/il/OMRNode.cpp

```diff
@@ -8642,3 +8642,55 @@ OMR::Node::resetFlagsForCodeMotion()
 /**
  * Node flags functions end
  */
+
+/**
+ * \brief
+ *    Return true of the node is a load of static final field
+ */
+bool
+OMR::Node::isLoadOfStaticFinalField()
+   {
+   if (self()->getOpCode().isLoadDirect() &&
+       self()->hasSymbolReference())
+      {
+      TR::Symbol *sym = self()->getSymbol();
+      if (sym->isFinal() &&
+          sym->isStaticField())
+         return true;
+      }
+   return false;
+   }
+
+/**
+ * \brief
+ *    Return true of the node is a call with osrFearPointSymbol
+ */
+bool
+OMR::Node::isOSRFearPoint()
```

Comment:
Since this is only checking for calls to the special OSRFearPoint helper I think this needs a better name because this sounds very generic when this is actually very specific.